### PR TITLE
Upgrade to marked.js for markdown rendering.

### PR DESCRIPTION
**Pull request by kadams54**
Our previous library (node-markdown) was deprecated in favor of marked. Marked also supports some of the things (like Github Flavor Markdown) that we need to bring README rendering on par with other uses of Markdown (e.g., the docs). Finally, it should enable syntax highlighting, in conjunction with https://github.com/juju/juju-gui/pull/1290.

### CODE REVIEW

**Comment by kadams54** [comment]
## QA

You may need to run `make clean-gui fast-babel gui` first. Pull up `/services/?store=trusty%2Fwordpress` and verify that the README rendered. Look at a few other charms and bundles in the details view and verify that they all render OK.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4296/
Test PASSed.

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "juju-gui",
-  "version": "1.9.7",
+  "version": "2.0.0",
```
Comment (makyo) [inline npm-shrinkwrap.json]: Is this done programmatically?

**Comment by makyo** [comment]
:+1: no QA yet

File: npm-shrinkwrap.json (hunk 1 above)
Comment (kadams54) [inline npm-shrinkwrap.json]: `npm shrinkwrap` did that, so I'm assuming it's pulling the version set in `package.json`.

**Comment by makyo** [comment]
QA :OK:

**Comment by anthonydillon** [comment]
Code and QA :+1:

**Comment by hatched** [comment]
Nice! :+1: :ok:
I see that it solves a number of the rendering issues with readme's that were in the previous release. GW.

:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/1472

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**